spec/support/test_helpers/teacher_training_public_api_helper.rb

```diff
@@ -27,20 +27,20 @@ def stub_teacher_training_api_provider(provider_code:, recruitment_cycle_year: R
     )
   end
 
-  def stub_teacher_training_api_course_with_site(recruitment_cycle_year: RecruitmentCycle.current_year, provider_code:, course_code:, site_code:, vacancy_status: 'full_time_vacancies', course_attributes: [], site_attributes: [])
-    course_attributes = course_attributes.any? ? [course_attributes.first.merge(code: course_code)] : [{ code: course_code }]
+  def stub_teacher_training_api_course_with_site(recruitment_cycle_year: RecruitmentCycle.current_year, provider_code:, course_code:, site_code:, vacancy_status: 'full_time_vacancies', course_attributes: [], site_attributes: [], course_uuid: SecureRandom.uuid)
+    course_attributes = course_attributes.any? ? [course_attributes.first.merge(code: course_code, uuid: course_uuid)] : [{ code: course_code, uuid: course_uuid }]
```

Comment:
Would it be better practice to modify the fixtures file rather than amending the stubbing method?

Reply:
Yes. The fixture should reflect what we receive from the TTAPI so it should be updated.

Reply:
Ok great, will implement